hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpMessageDispatcherService.java

```diff
@@ -190,4 +214,8 @@ private Artifact convertArtifact(final Target target, final LocalArtifact localA
         return artifact;
     }
 
+    public void setServiceMatcher(final ServiceMatcher serviceMatcher) {
```

Comment:
This is only required for tests so reduce visibility to package protected. It is not intendet to set this manually.

`void setServiceMatcher(final ServiceMatcher serviceMatcher)`